catalog/SearchBarScreen.tsx

```diff
@@ -1,18 +1,124 @@
-import React, { useState } from 'react'
```

Comment:
@J-Rehan The `SearchBarScreen` would be where we would put the `SearchBar` component. Let's just rename this file to be `GameCardScreen` so it corresponds to the components it's demoing.